### PR TITLE
Add support for python 3+

### PR DESCRIPTION
**Pull request by ehysi**

### CODE REVIEW

**Review by MayeulRousselet — CHANGES_REQUESTED** [review]
Missing a CHANGELOD.md

File: plugin.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "id": "gdpr",
-    "version": "0.1.6",
+    "version": "0.2.0",
```
Comment (alexbourret) [inline plugin.json:3]: The plugin version has to be updated in the Makefile as well

Comment (ehysi) [reply plugin.json:3]: Fixed in [0fe9fca](https://github.com/dataiku/dss-plugin-gdpr/pull/1/commits/0fe9fcad0ad7b1168226091f0cb6a5e093991ca1)

**Comment by ehysi** [comment]
> Missing a CHANGELOD.md

Added in [0fe9fca](https://github.com/dataiku/dss-plugin-gdpr/pull/1/commits/0fe9fcad0ad7b1168226091f0cb6a5e093991ca1)

**Comment by ehysi** [comment]
Just tested the Java part. It compiles correctly and we can enter gdpr data in the flow/datasets.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+## [Version 0.2.0](https://github.com/dataiku/dss-plugin-gdpr/releases/tag/v0.2.0) - Feature release - 2023-11-27
+
+- Add support for Python 3
+
+## Version 0.1.6 - Feature release - 2023-06-27
```
Comment (MayeulRousselet) [inline CHANGELOG.md]: Stricto sensu la 1ere release avait été faite il y bien longtemps...dater à la vraie date?

**Review by MayeulRousselet — CHANGES_REQUESTED** [review]

**Review by MayeulRousselet — APPROVED** [review]

**Review by alexbourret — APPROVED** [review]

**Pull request merged**